forui/test/src/widgets/accordion/accordion_controller_test.dart

```diff
@@ -0,0 +1,363 @@
+import 'package:flutter/animation.dart';
+import 'package:flutter/widgets.dart';
+
+import 'package:flutter_test/flutter_test.dart';
+import 'package:mockito/annotations.dart';
+import 'package:mockito/mockito.dart';
+
+import 'package:forui/forui.dart';
+import 'accordion_controller_test.mocks.dart';
+
+@GenerateNiceMocks([MockSpec<AnimationController>()])
+@GenerateNiceMocks([MockSpec<Animation>()])
+void _setup(List<AnimationController> animationControllers, List<Animation<int>> animations, int length) {
+  animations.clear();
+  animationControllers.clear();
+
+  for (int i = 0; i < length; i++) {
+    animationControllers.add(MockAnimationController());
+    animations.add(Tween(begin: 0, end: 100).animate(animationControllers[i]));
+    when(animationControllers[i].forward()).thenAnswer((_) {
+      when(animationControllers[i].value).thenReturn(1.0);
+      return TickerFuture.complete();
+    });
+
+    when(animationControllers[i].reverse()).thenAnswer((_) {
+      when(animationControllers[i].value).thenReturn(0.0);
+      return TickerFuture.complete();
+    });
+  }
+}
+
+void _tearDown(List<AnimationController> animationControllers, length) {
+  for (int i = 0; i < length; i++) {
+    animationControllers[i].dispose();
+  }
+}
```

Comment:
Why do we need a length for this? Can we just iterate over animation controllers?